.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
Do we need this `.env` file? What is the use of it?

Reply:
create-react-app having some dependencies graph issue. so in some machines it won't run npm start by keeping SKIP_PREFLIGHT_CHECK=true it help us to ignore this issue and run.
https://github.com/facebook/create-react-app/issues/5972